mlrun/runtimes/__init__.py

```diff
@@ -29,24 +27,68 @@
 )  # noqa
 from .constants import MPIJobCRDVersions
 from .nuclio import nuclio_init_hook
-from .serving import MLModelServer
+from .serving import ServingRuntime, new_v2_model_server
+
+# for legacy imports (MLModelServer moved from here to /serving)
+from ..serving import MLModelServer  # noqa
 from .sparkjob import SparkRuntime, SparkRuntimeHandler  # noqa
 from mlrun.runtimes.utils import resolve_mpijob_crd_version
 
 
+def new_model_server(
```

Comment:
@Hedingber this is also for backwards compatibility, people today import the `mlrun.runtimes.new_model_server` , note that for v2 models they dont have to use `new_model_server` since u have `func.add_model()` and other config methods

Reply:
You can solve backwards compatibility with importing it here, this code makes much more sense sitting there